index.html

```diff
@@ -37,190 +44,191 @@ <h2>Software Developer</h2>
+          </div>
+        </div>
+        <div class="work-container">
+          <div class="project-img"></div>
+          <div class="project-details">
+            <h2>Multi-post stories</h2>
+            <p>A daily selection of privately personalized reads;<br>
+              no accounts or sign-ups required. has been the <br>
+              industry's standard dummy text ever since the <br>
+              1500s, when an unknown printer took a standard 
+              dummy text.</p>
+            <ul class="programming-languages">
+              <li>JS</li>
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>Ruby</li>
+            </ul> <button type="button" class="btn" id="1">See project</button>
+          </div>
+        </div>
+        <div class="work-container">
+          <div class="project-img"></div>
+          <div class="project-details">
+            <h2>Multi-post stories</h2>
+            <p>A daily selection of privately personalized reads;<br>
+              no accounts or sign-ups required. has been the <br>
+              industry's standard dummy text ever since the <br>
+              1500s, when an unknown printer took a standard 
+              dummy text.</p>
+            <ul class="programming-languages">
+              <li><a href="">CSS</a></li>
+              <li><a href="">html</a></li>
+              <li><a href="">bootstrap</a></li>
+              <li><a href="">Ruby</a></li>
+            </ul> <button type="button" class="btn" id="1">See project</button>
+          </div>
+        </div>
+
+      </section>
+
+
+      <section class="contact-form">
+        <h2 class="contact-us">Let’s chat!</h2>
+        <p class="paragraph3">If you have an application you are
+          interested in developing, a feature that you <br>
+          need built or a project that needs coding.
+          I'd love to help with it </p>
+          <ul>
+            <li class="full name">
+              <input type="text" id="jeremy" name="user_name" />
+           <!-- </li>
+            <li> -->
+              <input type="text" id="lastname" name="last_name" />
+            </li>
+
+            <li>
+              <input type="email" id="mail" name="user_email" />
+            </li>
+            <li>
+              <textarea id="msg" name="user_message"></textarea>
+            </li>
+          </ul>
+
+          <li class="button">
+            <button type="button" class="btn" id="1">Send</button>
+          </li>
+        </form>
+      </section>
+   
+      <footer class="section-footer">
+        <ul class="social-icons">
+          <li><a href=""><img src="./img/Vector.png" alt="github"></a></li>
+          <li><a href=""><img src="./img/Frame.png" alt="Linkedin"></a></li>
+          <li><a href=""><img src="./img/Vector5.png" alt="angel"></a></li>
+          <li><a href=""><img src="./img/twitter.png" alt="twitter"></a></li>
+          <li><a href=""><img src="./imG/M.png" alt="medium"></a></li>
```

Comment:
Kindly update the path of your medium icon to './img/M.png' instead of './imG/M.png' 😉:
![image](https://user-images.githubusercontent.com/42799579/199631756-dff5b3bc-1a54-4af4-8629-f16d79b98245.png)

Reply:
![imagen](https://user-images.githubusercontent.com/90804443/199645706-9f09e891-a876-4307-b2b8-f09ad2e993ce.png)
why i can see it in my google , and when i submite it it looks like that ?

Reply:
> ![imagen](https://user-images.githubusercontent.com/90804443/199645706-9f09e891-a876-4307-b2b8-f09ad2e993ce.png) why i can see it in my google , and when i submite it it looks like that ?

🤔 I'm not sure how you are seeing this icon correctly, but the image path is misspelled. Maybe your computer autocorrects it for you. The G in 'img' should be lower case, like 'img' 'not imG'